shared/test/test_net_sim_api.rb

```diff
@@ -0,0 +1,55 @@
+require 'minitest/autorun'
+require 'rack/test'
+require File.expand_path '../../../deployment', __FILE__
+require File.expand_path '../../middleware/net_sim_api', __FILE__
+
+ENV['RACK_ENV'] = 'test'
+
+class NetSimApiTest < Minitest::Unit::TestCase
+
+  def create_record(record)
+    @net_sim_api.post "/v3/netsim/#{@shard_id}/#{@table_name}", record.to_json, 'CONTENT_TYPE' => 'application/json;charset=utf-8'
```

Comment:
Is a helper available for constructing these URLs?  I'm less concerned with the test code but for other production code it would be nice not to scatter knowledge of the format in case we decide to change it.